CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Internal: Added test suite
```

Comment:
```suggestion
- Add unit tests
```

Reply:
<img width="812" alt="Screenshot 2022-10-10 at 11 08 30" src="https://user-images.githubusercontent.com/10109428/194832693-0404fca9-c812-4f00-9f40-11c1cf87f32a.png">
@mariux i guess not then?